weather_dl/download_pipeline/clients.py

```diff
@@ -53,6 +53,11 @@ def num_requests_per_key(self, dataset: str) -> int:
         """Specifies the number of workers to be used per api key for the dataset."""
         pass
 
+    @abc.abstractmethod
+    def get_license_url(self) -> str:
```

Comment:
Since this takes no arguments in, consider making it an abstract property.

Reply:
If this is a property, you can omit the `get` language. That's a less pythonic naming pattern.

Reply:
If this is a property, you can omit the `get` language. That's a less pythonic naming pattern.

Reply:
Implemented abstract property for license_url.